processing/src/main/java/io/druid/query/select/SelectQueryEngine.java

```diff
@@ -59,6 +60,9 @@
       );
     }
 
+    // this is safe (see ServerManager)
```

Comment:
this is safe because at teh point where this code is called, only one datasource should exist. Can we update the comment?